pkg/scheduler/framework/runtime/registry.go

```diff
@@ -43,3 +47,41 @@ func (r Registry) Merge(in Registry) error {
 	}
 	return nil
 }
+
+// FactoryNames returns all known plugin names
+func (r Registry) FactoryNames() []string {
+	return sets.StringKeySet(r).List()
+}
+
+// Filter out the disabled plugin
+func (r Registry) Filter(names []string) Registry {
+	var retRegistry = make(Registry)
+	for _, name := range names {
+		// --plugins=*
+		if name == "*" {
+			for name, factory := range r {
+				klog.V(1).Infof("Starting Scheduler plugin %q", name)
```

Comment:
I'd prefer the log can be dumped, since it only prints once, right? --> `klog.Infof()`

By the way, do you think `Enable` is more appropriate than `Starting`?